src/actions/arcActions.ts

```diff
@@ -338,7 +340,7 @@ async function getProposalDetails(daoInstance: Arc.DAO, votingMachineInstance: A
     reputationChange: Util.fromWei(contributionProposal.reputationChange),
     periodLength: Number(contributionProposal.periodLength),
     preBoostedVotePeriodLimit,
-    proposer: proposalDetails.proposer,
```

Comment:
onProposalExecuted also updates the proposer, so this needs to happen there too

Reply:
https://github.com/daostack/alchemy/pull/164/commits/23795c8413e09bfc63ca4f9a36355bc069811917